planning/behavior_velocity_planner/src/scene_module/detection_area/debug.cpp

```diff
@@ -55,7 +58,8 @@ geometry_msgs::msg::Point toMsg(const lanelet::BasicPoint3d & point)
 }
 
 visualization_msgs::msg::MarkerArray createCorrespondenceMarkerArray(
-  const lanelet::autoware::DetectionArea & detection_area_reg_elem, const rclcpp::Time & now)
+  const lanelet::autoware::DetectionArea & detection_area_reg_elem, const rclcpp::Time & now,
+  ColorRGBA marker_color)
```

Comment:
I think adding reference to ```marker_color``` argument is better use
```suggestion
  const lanelet::autoware::DetectionArea & detection_area_reg_elem, const rclcpp::Time & now,
  ColorRGBA & marker_color)
```